### ISSUE TITLE
Letters mistaken for ordinals?

### ISSUE THREAD

**Issue by sjenset** [nystudio107/craft3-typogrify#21]
Ran into a weird issue today - unfortunately, I'm not sure exactly when it started appearing (bug-report filed by a customer last night, but based on an article that's ~2 weeks old). This is on a Norwegian site, so I guess it might have something to do with transliteration - but not sure.

Basically, I've had to turn `set_smart_ordinal_suffix`from `true` to `false` to prevent the plugin for wrapping the letter `e` in a `<sup class="ordinal">` if it directly follows an uppercase `D`:

![image](https://user-images.githubusercontent.com/12337525/51894557-66f01a00-23a8-11e9-9341-48f21dc0251a.png)

Both `set_diacritic_language` and `set_hyphenation_language` are set to `nb-NO` - other than that I think the only config-tweak that has been made is to disable `set_smart_fractions` - but let me know if you need any more settings etc. to look into this :)

**Comment by mundschenk-at** [comment]
You have to disable Roman numeral detection.

**Comment by sjenset** [comment]
> You have to disable Roman numeral detection.

What's the setting for that called?

**Comment by mundschenk-at** [comment]
It's `\PHP_Typography\Settings::set_smart_ordinal_suffix_match_roman_numerals` in the library. Not sure how that maps to craft-typography. Should be off by default, though.

**Comment by sjenset** [comment]
> It's `\PHP_Typography\Settings::set_smart_ordinal_suffix_match_roman_numerals` in the library. Not sure how that maps to craft-typography. Should be off by default, though.

Hmm, this is weird. I've never set it to "on" in any way to the best of my knowledge, but when I went back to re-examine this issue now (I had, tbh, forgotten about it slightly) the issue seems to have disappeared even though I've re-enabled `set_smart_ordinal_suffix` locally.

Maybe a underlying Craft-update did something weird in one of the minor-versions that has since been fixed? 😕
